src/clinical/donor-repo.ts

```diff
@@ -392,7 +393,17 @@ export const donorDao: DonorRepository = {
       submitterId: { $in: submitterIds },
       programId: programId,
     });
-    const mapped = result.map((d: DonorDocument) => {
+    const mapped = result.map(d => {
+      return MongooseUtils.toPojo(d) as Donor;
+    });
+    return F(mapped);
```

Comment:
what on earth is `F`!?

...I know it's just the established pattern, and it's obviously a function, but what does it do!?
without having to change the whole app, at least the import could be renamed in this file.

Reply:
rewrites the type as `DeepReadonly<T>`. Again, following the service convention, I do not recommend adopting this pattern elsewhere.

Reply:
`import { F as Freeze }` should help... but even then, what's `freezing`? 😮‍💨

Reply:
can't update/extend properties dynamically
if I remember correctly it's a very light lib around the native Object.freeze() https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/freeze